redbot/core/core_commands.py

```diff
@@ -1152,10 +1155,18 @@ def get_bot_uptime(self, *, brief: bool = False):
             else:
                 if pred.result is True:
                     await ctx.send(_("OK, it's on its way!"))
-                    async with ctx.author.typing():
-                        await ctx.author.send(
-                            _("Here's a copy of the backup"), file=discord.File(str(backup_file))
+                    try:
+                        async with ctx.author.typing():
+                            await ctx.author.send(
+                                _("Here's a copy of the backup"),
+                                file=discord.File(str(backup_file)),
+                            )
+                    except discord.Forbidden:
+                        await ctx.send(
+                            _("I don't seem to be able to DM you. Do you have closed DMs?")
                         )
+                    except discord.HTTPException:
+                        await ctx.send(_("Sending the backup file failed. It may be too large."))
```

Comment:
```suggestion
                        await ctx.send(_("I could not send the file. It is still available at {location}").format(backup_file))
```
This should serve as a better catch-all since we no longer expect this to be a size issue, but there are other things such as API stability which could cause this.

Reply:
Yeah you're right, I'm just gonna leave it as an open-ended "I could not send the file" rather than try to guess the issue that occurred